mobile/src/main/java/com/opendashcam/BackgroundVideoRecorder.java

```diff
@@ -157,9 +157,19 @@ private void initMediaRecorder(final SurfaceHolder surfaceHolder) {
 
         mediaRecorder = new MediaRecorder();
         mediaRecorder.setCamera(camera); // TODO See if we can remove this line. We can't, because media recorder should know what camera object will be used
-        mediaRecorder.setAudioSource(MediaRecorder.AudioSource.CAMCORDER);
+        
         mediaRecorder.setVideoSource(MediaRecorder.VideoSource.CAMERA);
-        mediaRecorder.setProfile(camcorderProfile);
+
+        if (settings.getBoolean("record_audio", true)) {
+            mediaRecorder.setAudioSource(MediaRecorder.AudioSource.CAMCORDER);
+            mediaRecorder.setProfile(camcorderProfile);
+        } else {
+            mediaRecorder.setOutputFormat(MediaRecorder.OutputFormat.MPEG_4);
```

Comment:
With the `else` predicate here, does it mean that the video parameters will be set only when `record_audio` is `false`?
Don't we need them for both _with_ and _without_ audio? Need to check this.